### DISCUSSION TITLE
Release Date of 3.35.0?

### DISCUSSION THREAD

**Discussion by sasacocic** [xerial/sqlite-jdbc#616] [Q&A] [answered]
Just curious is there a timeline for when version 3.35.0 will be released? I'm assuming that `sqlite-jdbc` versions correspond to SQLite versions somehow. 

Anyways there's lots of features in 3.35.0 that would be very useful for me, and I was just wondering if there's a release date scheduled or an approximation of when it will be released!

**Comment by xerial** [comment]
I've also found 3.35.0 is useful (e.g., RETURNING syntax support) 
sqlite-jdbc 3.35.0.1 and 3.36.0 based on the latest SQLite version have been released.

There has been no fixed released date nor schedule for sqlite-jdbc. It totally depends on my personal interest and my free time. This might be one of the tips to keep maintaining OSS project for more than 10 years. It's also quite similar to the development of the original SQLite. Some times, there is no update for more than 6+ months 😅 

The development of SQLite becomes active recently https://sqlite.org/chronology.html. Let me know (here or at Twitter) if you notice new interesting changes.

**Comment by xerial** [comment, marked as answer]
3.36.0 has been released
